VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1169,7 +1186,9 @@ subroutine vom_hourly_init ()
       use vom_vegwat_mod
       implicit none
 
-      INTEGER :: ii
+      INTEGER :: ii           !counter
+      REAL*8 :: Ma_lg(3)      !fraction of absorbed radiation grasses
+      REAL*8 :: Ma_lt(3)      !fraction of absorbed radiation trees
```

Comment:
The way it is used below, this is the fraction of absorbed radiation per crown area.

Reply:
Yes, true, will document that.